packages/wallets/src/components/WalletsResetMT5Password/WalletSuccessResetMT5Password.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC, useCallback } from 'react';
+import useDevice from '../../hooks/useDevice';
+import MT5SuccessPasswordReset from '../../public/images/mt5-success-password-reset.svg';
+import { ModalStepWrapper, WalletButton } from '../Base';
+import { useModal } from '../ModalProvider';
+import { WalletsActionScreen } from '../WalletsActionScreen';
+
+type WalletSuccessResetMT5PasswordProps = {
+    title: string;
+};
+
+const WalletSuccessResetMT5Password: FC<WalletSuccessResetMT5PasswordProps> = ({ title }) => {
+    const { hide } = useModal();
+    const { isDesktop, isMobile } = useDevice();
+
+    const renderFooter = useCallback(() => {
+        return isMobile ? <WalletButton isFullWidth onClick={() => hide()} size='lg' text='Done' /> : null;
+    }, [isMobile, hide]);
+
+    const renderButtons = useCallback(() => {
+        return isDesktop ? <WalletButton onClick={() => hide()} size='lg' text='Done' /> : <></>;
```

Comment:
@aizad-deriv maybe better to use null instead of <></>?

Reply:
I tried that before with `null` it will give me a typescript error because of the typing of the component.